apis/v1/controllers/addresses_controller.js

```diff
@@ -120,8 +120,8 @@ exports.makeAddressReadable = (addresses) => {
                 let newAddress = {
                     ...doc,
                     status: statusKey,
-                    parish: doc.parish.parishName,
-                    user_id: doc.user_id.email,
+                    parish: doc.parish?.parishName,
```

Comment:
Dont think these work in javascript.
Tested it with invalid data and it broke
why not use a ternary operator to check if the parish is truthy.

Reply:
and for the user_id too